zynq_boot_bin/build_boot_bin.sh

```diff
@@ -2,12 +2,12 @@
 set -ex
 
 XSA_FILE=$1
-UBOOT_FILE=$2
```

Comment:
You need to let u-boot file argument as optional - users come many times with their own u-boot.